attest/api/Cargo.toml

```diff
@@ -1,15 +1,19 @@
 [package]
 name = "attest-api"
-version = "0.1.0"
+version = "0.1.1"
 authors = ["MobileCoin"]
 license = "MIT/Apache-2.0"
 edition = "2018"
 description = "gRPC APIs for encrypted communications with an enclave"
 keywords = ["SIGMA", "Cryptography", "Key Exchange", "Diffie-Hellman", "SGX", "Attestation"]
 readme = "README.md"
+links = "mc-attest-api"
```

Comment:
Why is this needed?

Reply:
You need a `links` key in `Cargo.toml`, in order to have cargo export variables. So, you do `cargo_emit::pair!` to emit the "cargo:KEY=VALUE" stuff, and if you have a `links` key, dependents of the crate doing the emission will get `DEP_<SCREAMING_SNAKE_LINKS>_KEY=VALUE` injected into their environment.

Reply:
https://doc.rust-lang.org/cargo/reference/build-scripts.html#the-links-manifest-key

Reply:
lol SCREAMING_SNAKE_LINKS